src/core.js

```diff
@@ -159,3 +149,8 @@ jQuery.fn.size = function() {
 	migrateWarn( "jQuery.fn.size() is deprecated; use the .length property" );
 	return this.length;
 };
+
+jQuery.parseJSON = function() {
+	migrateWarn( "jQuery.parseJSON is deprecated; use JSON.parse" );
```

Comment:
Well, if Migrate 3 is supposed to bring back only deprecated/removed behavior that was present in jQuery 1.12/2.2, then `jQuery.parseJSON` in those versions is basically `JSON.parse` (well, with one workaround for Android 2.3 but jQuery 3.0 doesn't support it in that capacity so it doesn't matter) and that's the same hook that's used by the ajax json converter. So it doesn't seem that we should continue to patch it in Migrate 3.